config.toml

```diff
@@ -442,15 +441,16 @@ language_alternatives = ["en"]
 
 [languages.hi]
 title = "Kubernetes"
-description = "Production-Grade Container Orchestration"
-languageName = "हिन्दी (Hindi)"
+description = "प्रोडक्शन-ग्रेड कंटेनर ऑर्केस्ट्रेशन"
+languageName = "हिन्दी"
 languageNameLatinScript = "Hindi"
 weight = 12
 contentDir = "content/hi"
 languagedirection = "ltr"
 
 [languages.hi.params]
-time_format_blog = "01.02.2006"
+time_format_blog = "02.01.2006"
+# A list of language codes to look for untranslated content, ordered from left to right.
```

Comment:
Should we update these lines in `dev-1.22-hi.1` so that they are the same as `main`?